src/main/webapp/dev/js/common/instructor.js

```diff
@@ -358,14 +370,17 @@ function bindCourseDeleteAllLinks() {
         event.preventDefault();
 
         const $clickedLink = $(event.currentTarget);
-        const messageText = 'Are you sure you want to permanently delete all the courses? '
-                + 'This operation will delete all students and sessions in these courses. '
-                + 'All instructors of these courses will not be able to access them hereafter as well.';
+        const softDeletedCourses = $('[id^="recoverycourseid"]').toArray()
+                .map(softDeletedCourseIdElement => $(softDeletedCourseIdElement).data('softDeletedCourseId'));
+        const message = `<p>Are you sure you want to permanently delete the following ${softDeletedCourses.length} courses `
+                .concat('in the <b>Recycle Bin</b>? This operation will delete all students and sessions in these courses. ')
```

Comment:
Why is this suddenly changed to `concat`? This is very awkward and not better than the string concatenation operator?

Keep to using template literals and breaking it up into multiple lines appropriately ([Line length style guide](https://github.com/airbnb/javascript#strings--line-length)).

For example, something like this:
```Javascript
const message = `<p>Are you sure you want to permanently delete the following ${softDeletedCourseIds.length} courses in the <b>Recycle Bin</b>? This operation will delete all students and sessions in these courses. All instructors of these courses will not be able to access them hereafter as well.</p>
${generateUnorderedListHtml(softDeletedCourseIds)}`;
```

We need to change our ESLint rules though (`teammates-eslint.yml`) because we overridden the base rule:
```
max-len: ["error", 125, 2, {
    "ignoreUrls": true,
    "ignoreComments": false,
    "ignoreRegExpLiterals": true,
    "ignoreStrings": true,
    "ignoreTemplateLiterals": true
}]
```

Reply:
Are you suggesting that we need to keep it in one line? The thing is that `npm run lint` will throw error if there's over 125 chars in a line. And I tried to keep `+`, but it throws an error since we are concatenating a function (`generateUnorderedListHtml`).

Reply:
Oh so you mean we just change `teammates-eslint.yml`?

Reply:
Yeah, we were adapting the Airbnb rules anyway, it's just that when we overrode it to fit 125 characters we inadvertently clear some rules unintentionally.